components/org.wso2.carbon.identity.oauth.dcr/src/main/java/org/wso2/carbon/identity/oauth/dcr/DCRMConstants.java

```diff
@@ -95,4 +96,92 @@ public static class ErrorCodes {
     }
 
     public static final String OAUTH2 = "oauth2";
+
+    public static final String DCR_CONFIG_RESOURCE_TYPE_NAME = "DCR_CONFIGURATION";
+
+    public static final String DCR_CONFIG_RESOURCE_NAME = "TENANT_DCR_CONFIGURATION";
+
+    public static final String ENABLE_FAPI_ENFORCEMENT = "enableFapiEnforcement";
+    public static final String CLIENT_AUTHENTICATION_REQUIRED = "clientAuthenticationRequired";
+    public static final String SSA_JWKS = "ssaJwks";
+    public static final String MANDATE_SSA = "mandateSSA";
+
+    /**
+     * Enum for DCR Configuration related error messages.
+     */
+    public enum DCRConfigErrorMessage {
+
+        /**
+         * Unable to retrieve DCR configuration.
+         */
+        ERROR_CODE_DCR_CONFIGURATION_RETRIEVE("65020",
+                "Unable to retrieve DCR configuration.",
+                "Server encountered an error while retrieving the " +
+                        "DCR configuration of %s."),
```

Comment:
Should be able to merge with the previous line. New line is not needed.

Reply:
Fixed with [bcbf429](https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/2344/commits/bcbf429f60dda15052023305a458026998d9afa3)